Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -247,11 +247,11 @@ class BoolSetting: Setting, FeatureFlagsProtocol {
         settingDidChange?(control.isOn)
 
         if let featureFlagName = featureFlagName {
-            guard let key = featureFlags.featureKey(for: featureFlagName) else { return }
             TelemetryWrapper.recordEvent(category: .action,
                                          method: .change,
                                          object: .setting,
-                                         extras: ["pref": key as Any, "to": control.isOn])
+                                         extras: ["pref": featureFlagName.rawValue as Any,
```

Comment:
Checked with @dnarcese and she was fine with this being updated to this new  value.